backend/YouTubeV2.Application/Services/VideoServices/VideoService.cs

```diff
@@ -48,5 +48,27 @@ public async Task SetVideoProcessingProgressAsync(Video video, ProcessingProgres
             video.ProcessingProgress = processingProgress;
             await _context.SaveChangesAsync(cancellationToken);
         }
+
+        public async Task<VideoMetadataDto> GetVideoMetadataAsync(Guid id, CancellationToken cancellationToken = default)
```

Comment:
i think getting the videometadata should update the viewcount. I have no other idea how to do it differently